cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/SlangTestRunner.java

```diff
@@ -134,11 +156,28 @@ public Long trigger(String testCaseName, CompilationArtifact compilationArtifact
         String executionResult = executionReturnValues.getResult();
 
         String errorMessageFlowExecution = testsEventListener.getErrorMessage();
-        if (StringUtils.isNotEmpty(errorMessageFlowExecution) || !executionResult.equals(result)) {
-            // exception occurred during flow execution
-            throw new RuntimeException(errorMessageFlowExecution);
+
```

Comment:
You can add here the logging of the status of the tests, instead of the printing method in the event listener